packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSearchBar/CWSearchBar.tsx

```diff
@@ -82,7 +94,7 @@ export const CWSearchBar: FC<SearchBarProps> = ({
     // fire before resetting the search bar
     if (!resetTimer) {
       resetTimer = setTimeout(() => {
-        resetSearchBar();
+        // resetSearchBar();
```

Comment:
maybe remove?

Reply:
no no, this should be back, thanks for pointing out